config/initializers/phone.rb

```diff
@@ -1,24 +1,25 @@
-require_dependency 'barong/mock_sms'
-
 Barong::App.define do |config|
   config.set(:twilio_phone_number, '+15005550000')
   config.set(:twilio_account_sid, '')
   config.set(:twilio_auth_token, '')
+  config.set(:twilio_service_sid, '')
   config.set(:sms_content_template, 'Your verification code for Barong: {{code}}')
 end
 
 sid = Barong::App.config.twilio_account_sid
 token = Barong::App.config.twilio_auth_token
+service_sid = Barong::App.config.twilio_service_sid
 
 if sid == '' || token == ''
   if Rails.env.production?
     Rails.logger.fatal('No Twilio sid or token')
     raise 'FATAL: Twilio setup is invalid'
   end
-  client = Barong::MockSMS.new(sid, token)
 else
   client = Twilio::REST::Client.new(sid, token)
+  service = client.verify.services.create(friendly_name: Barong::App.config.app_name) unless service_sid.present?
 end
 
-Barong::App.define { |c| c.set(:sms_sender, client) }
+Barong::App.set(:twilio_client, client) if client
```

Comment:
and what will happen if not `client`?
We will have errors in the project?

Better we think about this now